app/products/playbooks/types/database/models/playbook_run_attribute_value.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import type PlaybookRunModel from './playbook_run';
+import type PlaybookRunAttributeModel from './playbook_run_attribute';
+import type {Relation} from '@nozbe/watermelondb';
+import type Model from '@nozbe/watermelondb/Model';
+
+/**
+ * The PlaybookRunAttributeValue interface, which will have all the fields from the PLAYBOOK_RUN_ATTRIBUTE_VALUE table
+ */
+interface PlaybookRunAttributeValueModelInterface extends Model {
+
+    /** attributeId : The ID of the attribute this attribute value belongs to */
+    attributeId: string;
+
+    /** runId : The ID of the playbook run this attribute belongs to */
+    runId: string;
```

Comment:
this isn't strictly needed, as the attribute is already linked, but I think it will make observing cleaner when we only need to observe filtering by runId